use-cases/using-Intel-SGX-and-kata.md

```diff
@@ -0,0 +1,68 @@
+# Kata Containers with SGX
+
+- [Install Host kernel with SGX support](#install-host-kernel-with-sgx-support)
+- [Install Guest kernel with SGX support](#install-guest-kernel-with-sgx-support)
+- [Run Kata Containers with SGX enabled](#run-kata-containers-with-sgx-enabled)
+
+Intel® Software Guard Extensions (SGX) is a set of instructions that increases the security
+of applications code and data, giving them more protections from disclosure or modification.
+At the time of writing this document, SGX patches have not landed on the Linux kernel
+project and `cloud-hypervisor` 0.8.0 doesn't support SGX, so specific versions of `cloud-hypervisor`,
+guest and host kernel must be installed to enable SGX.
+
+## Install Host kernel with SGX support
+
+The following commands were tested on fedora 32, they might work on other distros too.
+
+```sh
+git clone --depth=1 https://github.com/intel/kvm-sgx
+pushd kvm-sgx
+cp /boot/config-$(uname -r) .config
+yes n | make oldconfig
+# In the following step, enable: INTEL_SGX and INTEL_SGX_VIRTUALIZATION
+make menuconfig
+make -j$(($(nproc)-1)) bzImage
+make -j$(($(nproc)-1)) modules
+sudo make modules_install
+sudo make install
+popd
+sudo reboot
+```
+
+_Note: Run: `mokutil --sb-state` to check whether secure boot is enabled, if so, you will need to sign the kernel._
+
+Once you have restarted your system with the new brand Linux Kernel with SGX support, run
```

Comment:
It might be worth moving this block to the start of the doc (as that will minimise the update when SGX support is more widespread):

```md
# Checking if SGX is enabled

Run the following command to check if SGX support is available...

If SGX is not enabled, continue to section ... .